monitoring/ingestion/global-dashboard.json

```diff
@@ -0,0 +1,632 @@
+{
+  "__inputs": [
+    {
+      "name": "DS_PROMETHEUS_SCALITY",
```

Comment:
For the Zenko-operator dashboard input validation to work, inputs name can be one of the following: "replicas", "job_ingestion_processor", "job_ingestion_producer", "DS_PROMETHEUS", "DS_LOKI", "namespace" or "zenkoName". If needed we can add new ones. Feel free to reuse the inputs names from the two existing dashboard, they passed the validation.